### DISCUSSION TITLE
基于 formily 2.x，实现一个渲染到 cli 界面的表单

### DISCUSSION THREAD

**Discussion by quanru** [alibaba/formily#2668] [Ideas]
需求是为了使用同一个 schema json，生成 web 表单和 cli 表单。
比如将 schema json 映射到 [enquirer](https://www.npmjs.com/package/enquirer#select-prompt)；
目前是自己实现 x-reactions 的逻辑，现在看来比较适合降级实现、且是一个 formily 2.x 的 schema 规则子集，求一点点建议~

**Comment by zFitness** [comment]
目前搞的咋样了
